src/virtual-machine.js

```diff
@@ -590,7 +595,8 @@ class VirtualMachine extends EventEmitter {
         const extensionPromises = Array.from(extensions.extensionIDs).map(async extensionID => {
             if (!extensionManager.isExtensionLoaded(extensionID)) await extensionManager.loadExtensionURL(extensionID);
             const instance = this.extensionManager.getExtensionInstance(extensionID);
-            instance?.["load"]?.(fullJSON); // TODO: Verify that this is okay to do on already loaded extensions
+            // Had to comment out below line -- fullJSON is not defined
```

Comment:
Interesting! Looks like I added the `fullJson` parameter: https://github.com/mitmedialab/prg-extension-boilerplate/commit/ed98b24b69121f5a8cb9cc656b473a3ac03e0e9d#diff-c1721b462c4ebdada7a3a2e1e72ae03e6f830b83d739aea5354ed43b9f0b1ea5R823

This would've been to accomplish the [custom save data](https://github.com/mitmedialab/prg-extension-boilerplate/tree/dev/extensions#saving-custom-data-for-an-extension) functionality. We will want to keep supporting this, so we'll either want to add back this parameter, or perhaps there's a more elegant place for this to happen.? Let's think and discuss!